aeon_sql.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+import "aeon_common.proto";
+
+package aeon;
+
+option go_package = "./pb";
+
+// SQL API to Aeon - a distributed database based on Tarantool.
+service AeonSQLService {
```

Comment:
Let's remove the `Aeon` prefix: `SQLService`, `CRUDService`. The package name should serve as a namespace.

Reply:
Fixed.